addresses/deployed/matic.json

```diff
@@ -1,18 +1,18 @@
 [
   {
-    "address": "0xe5d708eb7b377F0db822D64d138AE29747D43369",
+    "address": "0x2f5a51B09044754C7C8b1A9deE29E046b5432038",
     "name": "Mangrove"
   },
   {
-    "address": "0xb43B23e87e045e196e839880F8C2806fCccE4221",
+    "address": "0x09f8b6E32DB426aa477ad815544d0573FeFdBeEF",
     "name": "MgvOracle"
   },
   {
-    "address": "0x106b805d76Ea65F8aF0Bb4E9499a2CB69162C75A",
+    "address": "0xe71f73C7C0cfb02Cfa3F9810659CD23cE88FFB28",
```

Comment:
1) Deploy with the new reader?
2) Also deploy mangroveorderenriched
3) IMHO it should ideally be multisig person who approves this - they can confirm the addresses match.

Reply:
We will deploy later, and we may not make a release with those addresses, but it's good to share those on git (they are deployed).